app/bootstrap.php

```diff
@@ -30,4 +33,23 @@
 
 require INFECTION_COMPOSER_INSTALL;
 
+$isDebuggerDisabled = empty((string) getenv(\Infection\Php\XdebugHandler::ENV_DISABLE_XDEBUG));
+
+$xdebug = new \Infection\Php\XdebugHandler(new \Infection\Php\ConfigBuilder(sys_get_temp_dir()));
+$xdebug->check();
+unset($xdebug);
+
+if (PHP_SAPI !== 'phpdbg'
+    && !\defined('HHVM_VERSION')
```

Comment:
As for me we can drop support of HHVM. But this condition was there before me. I've just move it from one of command.

@borNfreee what do you think? Also it could be different PR or I can do it in this one

Reply:
yeah, we don't support hhvm, it can be removed